### PR TITLE
(2/7) RFC79: Make study_es_0_inc data pass validation

### PR DESCRIPTION
**Pull request by forus**
- document how to load `study_es_0` study
- Remove `SpringUtil.initDataSource();` from multiple classes as it was causing not used services to fail during initialisation
-  organise imports (remove unused, no wildcard imports) for the classes that lost `SpringUtil.initDataSource();` statement.
- include into the test data (`seed_mini.sql`) genes that are used by `study_es_0` data.

Please review https://github.com/cBioPortal/cbioportal-core/pull/43 next.

Please note, to avoid the costly process of rebasing all pull requests (PRs), all feedback will be incorporated into the final PR.

### CODE REVIEW

File: tests/test_data/study_es_0_inc/data_methylation_hm27.txt [hunk 1]
```diff
@@ -4,7 +4,6 @@ ACAP3	116983	0.022	0.681	0.790
 ATAD3A	55210	0.229	0.946	0.439
 ATAD3B	83858	0.885	0.707	0.664
 ATAD3C	219293	0.660	0.315	0.694
-#AURKAIP1	54998
```
Comment (pieterlukasse) [inline tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: historically study_es_0 has always passed validation. Were these lines also there? What changed?

File: tests/test_data/study_es_0_inc/data_cna_pd_annotations.txt
Comment (pieterlukasse) [inline tests/test_data/study_es_0_inc/data_cna_pd_annotations.txt:1]: why is this file needed?

**Review by pieterlukasse — APPROVED** [review]
Looks good to me 👍 
Just a few questions

File: tests/test_data/study_es_0_inc/data_methylation_hm27.txt (hunk 1 above)
Comment (forus) [reply tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: study_es_0 has not passed validation before this fix. Remember automatic tests do not use this data, but you must load this study manually to see whether everything works. That's how I missed it.

File: tests/test_data/study_es_0_inc/data_cna_pd_annotations.txt
Comment (forus) [reply tests/test_data/study_es_0_inc/data_cna_pd_annotations.txt:1]: to load data to `alteration_driver_annotation` in the wide CNA file format. The long format has all the data in a single data file instead.

File: tests/test_data/study_es_0_inc/data_methylation_hm27.txt (hunk 1 above)
Comment (pieterlukasse) [reply tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: I think there is a misunderstanding: "_es_0" stands for "exit status 0" meaning "passes validation". So I'm wondering what change caused es_0 to start failing.

Comment (forus) [reply tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: I think the python validation script did not like the comment to be in the middle of the file.
Please also note, it's not the study_es_0 study folder, but the incremental upload data set.

Comment (pieterlukasse) [reply tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: so `#AURKAIP1` is specific to the incremental folder (i.e. you made this addition) and is _not_ there in the original study_es_0 folder?

Comment (pieterlukasse) [reply tests/test_data/study_es_0_inc/data_methylation_hm27.txt:7]: update: Ruslan added this line. Now he is removing it.

**Review by pieterlukasse — APPROVED** [review]

**Review by haynescd — APPROVED** [review]
LGTM
